tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -3858,7 +3857,6 @@ private function init_payment_methods() {
 			Link_Payment_Method::class,
 			Affirm_Payment_Method::class,
 			Afterpay_Payment_Method::class,
-			Klarna_Payment_Method::class,
```

Comment:
Might want to restore this too? Or nah, since it's just in the tests?

Reply:
Ah, yeah, I can restore that one. Resolved in 78506698f